notebook_test.py

```diff
@@ -0,0 +1,108 @@
+import papermill as pm
+import argparse
+
+from os import path
+
+from utils import FileHandler
+from utils.enums import Venue
+
+
+def parse_args():
+    """
+    Parses the program arguments
+    Returns
+    -------
+    args
+    """
```

Comment:
Can you rename the notebook_test.py to something a little more descriptive?  Maybe like **_notebook_collection_test.py_**?  

Maybe rename the hoss_subsetter_test.ipynb to something more descriptive too?  Like **_hoss_collection_test.ipynb_**? 
Thanks!